server/server.js

```diff
@@ -22,6 +24,12 @@ const upload = multer({ dest: 'server/assets/uploads/' })
 const app = express()
 const port = process.env.PORT || 3000
 
+// Job starts when clock hits the 0 hour, the 0 minute
```

Comment:
Can't you also add this to to the events.js file?

Afterwards you can change the import
 ```JavaScript 
import getFestivalData from './middleware/cron/events.js'
``` 
to 
 ```JavaScript 
import './middleware/cron/events.js'
```
In order to start the cron job.